docs/specs/xref.yml

```diff
@@ -42,3 +42,92 @@ outputs:
     { "content": "<p>Link to <a href=\"https://docs.microsoft.com/en-us/dotnet/api/system.string\">String</a></p>\n"}
   build.manifest:
 ---
+# Define uid in conceptual, resolve it
+inputs:
+  docfx.yml: |
+    buildInternalXrefMap: true
+  docs/a.md: |
+    ---
+    title: Title from yaml header a
+    uid: a
+    ---
+  docs/b.md: |
+    ---
+    title: Title from yaml header b
+    uid: b
+    ---
+  docs/c.md: Link to @a
+outputs:
+  docs/c.json: | 
+    {"content":"<p>Link to <a href=\"docs/a.json\">Title from yaml header a</a></p>\n"}
+  docs/a.json:
+  docs/b.json:
+  build.manifest:
+---
+# Define uid in SDP, resolve it
+inputs:
+  docfx.yml: |
+    buildInternalXrefMap: true
+  docs/a.json: |
+    {
+      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+      "uid": "a",
+      "name": "test",
+      "fullName": "test.test",
+      "description": "a.description"
+    }
+  docs/c.md: Link to @a
+outputs:
+  docs/c.json: | 
+    {"content":"<p>Link to <a href=\"docs/a.json\">a</a></p>\n"}
+  docs/a.json:
+  build.manifest:
+---
+# Define duplicate uid in conceptual and SDP, should output warning and do not resolve it
```

Comment:
>Define duplicate uid in conceptual and SDP, should output warning and do not resolve it [](start = 2, length = 87)

what about choose one of the duplicate uids as the result? v2 should have this behavior.

Reply:
which one should be picked? could it be random?

---
In reply to: [219089394](https://github.com/dotnet/docfx/pull/3437#discussion_r219089394) [](ancestors = 219089394)

Reply:
I think so, maybe sort by href to make result stable

---
In reply to: [219090345](https://github.com/dotnet/docfx/pull/3437#discussion_r219090345) [](ancestors = 219090345,219089394)